### ISSUE TITLE
Feature request to use Intel Myriad chips for acceleration Depth Map from Stereo Images

### ISSUE THREAD

**Issue by AlexeyAB** [opencv/opencv#15167]
Feature request to use Intel Myriad chips for acceleration Depth Map from Stereo Images by using Myriad SHAVES cores - to accelerate OpenCV functions: https://docs.opencv.org/4.1.1/dd/d53/tutorial_py_depthmap.html

Currently, OpenCV can use Myriad X chips for Deep Learning tasks (via OpenVINO DL IE backend), but not for other tasks like Stereo Depth processing.

As known KEY BENEFITS of Myriad X chip is: **New Vision Accelerators including Stereo Depth** https://www.movidius.com/MyriadX But there are no examples or API for Stereo Depth processing by using OpenVINO in free access.

It would be nice to be able to use uniform OpenCV API for the basic functions of the Miriad X chip in conjunction with a very weak CPU (Intel Atom). Because Atom CPU doesn't have enough performance to get a Stereo-Depth-Map on its own with high FPS.

**Comment by dkurt** [comment]
Just for reference: https://luxonis.com/depthai and https://newsroom.intel.com/news/intel-realsense-stand-alone-inside-out-tracking-camera/

Key thing is a firmware. Most likely, OpenCV couldn't communicate with VPU directly but only through 3rdparty library.

**Comment by AlexeyAB** [comment]
@dkurt Yes, I think at first it should be implemented in one of the libraries (for example, OpenVINO).

**Comment by jsxyhelu** [comment]
@AlexeyAB To acceleration Depth Map from Stereo Images by OpenVINO sounds great! Is there any reference?

**Comment by AlexeyAB** [comment]
@jsxyhelu OpenVINO doesn't support Depth Map from Stereo Images yet.
